src/main_controller.ts

```diff
@@ -85,9 +87,9 @@ export class MainController implements vscode.Disposable {
         ];
 
         const workspaceFolder = vscode.workspace.workspaceFolders;
-        const cwd = workspaceFolder && workspaceFolder.length ? workspaceFolder[0].uri.fsPath : undefined;
-        if (cwd && !config.useWsl && !vscode.env.remoteName) {
-            args.push("-c", `cd ${cwd}`);
+        const cwd = workspaceFolder?.length ? workspaceFolder[0].uri.fsPath : undefined;
+        if (cwd && !vscode.env.remoteName) {
+            args.push("-c", `cd ${config.useWsl ? wslpath(cwd) : cwd}`);
```

Comment:
`cd` is somewhat clumsy, should we pass `cwd` to `spawn()`?  https://nodejs.org/api/child_process.html#child_processspawncommand-args-options

Reply:
Haha, I realized this after the merger. Too lazy to make changes.